pkg/koordlet/metricsadvisor/be_collector.go

```diff
@@ -75,7 +76,8 @@ func getBECPURealMilliLimit() (int, error) {
 	}
 	limit = len(cpuSet) * 1000
 
-	cfsQuota, err := koordletutil.GetRootCgroupCurCFSQuota(corev1.PodQOSBestEffort)
+	BECgroupParentDir := koordletutil.GetPodQoSRelativePath(corev1.PodQOSBestEffort)
+	cfsQuota, err := resourceexecutor.NewCgroupReader().ReadCPUQuota(BECgroupParentDir)
```

Comment:
make `getBECPURealMilliLimit` as a member function of collector struct, so it can reuse the cgroupReader of collector